snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -52,13 +52,16 @@ enum SnapshotError: Error, CustomDebugStringConvertible {
 }
 
 @objcMembers
+@MainActor
 open class Snapshot: NSObject {
     static var app: XCUIApplication?
     static var waitForAnimations = true
     static var cacheDirectory: URL?
     static var screenshotsDirectory: URL? {
         return cacheDirectory?.appendingPathComponent("screenshots", isDirectory: true)
     }
+    static var deviceLanguage = ""
+    static var currentLocale = ""
```

Comment:
It should be renamed to avoid conflicting `setLocale` implemented in this file between auto-generated by `@objcMembers`